### PR TITLE
GUVNOR-2340 - Annotation for element inside subprocess fails validation

### PR DESCRIPTION
**Pull request by tsurdilo**

### CODE REVIEW

**Comment by csadilek** [comment]
small indentation problem, otherwise looks good: https://github.com/tsurdilo/jbpm-designer/blob/master/jbpm-designer-backend/src/main/java/org/jbpm/designer/bpmn2/validation/BPMN2SyntaxChecker.java#L645

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/validation/BPMN2SyntaxCheckerTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package org.jbpm.designer.bpmn2.validation;
```
Comment (manstis) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/validation/BPMN2SyntaxCheckerTest.java]: Missing license header too!

**Comment by tsurdilo** [comment]
@csadilek @manstis Thanks guys! Amended

**Comment by csadilek** [comment]
master: 2d16f9b432f88276a632fdf933ddf518bfacbbde

**Pull request closed without merging**